Class/EvolizClient.php

```diff
@@ -48,7 +48,7 @@ public static function findOrCreate(Config $config, object $order): ?int
                         'iso2' => $order->get_billing_country(),
                         'addr' => $order->get_billing_address_1()
                     ],
-                    'phone' => $order->get_billing_phone(),
+                    'phone' => (string) $order->get_billing_phone(),
```

Comment:
Ce bug la je n'ai pas reussi à le reproduire... dans la doc de WP, la function get_billing_phone est censé renvoyer une string, mais on a un client qui a eu une erreur "Bad Request Error : The phone must be a string.". Donc j'ai ajouté un cast au cas ou mais je vois pas comment ça peut arriver